gc.rb

```diff
@@ -272,7 +271,7 @@ def self.using_rvargc?
   #    GC.measure_total_time = true/false
   #
   # Enable to measure GC time.
-  # You can get the result with `GC.stat(:time)`.
+  # You can get the result with <tt>GC.stat(:time)</tt>.
   # Note that the GC time measurement can introduce the performance regression.
```

Comment:
```suggestion
  # Note that GC time measurement can cause some performance overhead.
```

Would seem a bit more correct, no?